tools/iree-e2e-matmul-test.cc

```diff
@@ -226,6 +226,11 @@ REFERENCE_MATMUL(float, float, float, float)
 // [i32 <= i8 * i8 + i32]
 REFERENCE_MATMUL(int8_t, int8_t, int32_t, int32_t)
 
+// Reference mamtul instantiations from macro REFERENCE_MATMUL
+// for the int8_t input, int32_t accumlation, and int32_t result.
```

Comment:
update the comment: int8->int32

Reply:
Oops didn't see it before merging will send a NFC PR shortly